src/apollo/config.js

```diff
@@ -13,6 +13,8 @@ const wsHost =
     ? 'localhost:4142'
     : window.location.host;
 
+const wsProtocol = window.location.protocol === 'https:' ? 'wss' : 'ws';
```

Comment:
Need to use this on line 19.

Reply:
Oops the file wasn't saved and commited. Nice catch.